modules/deploying-lvms-on-sno-cluster.adoc

```diff
@@ -34,4 +34,43 @@ Before you deploy {lvms} on {sno} clusters, be aware of the following limitation
 
 * You can only create a single instance of the `LVMCluster` custom resource (CR) on an {product-title} cluster.
 * You can make only a single `deviceClass` entry in the `LVMCluster` CR.
-* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
\ No newline at end of file
+* When a device becomes part of the `LVMCluster` CR, it cannot be removed.
+
+[id="lvms-deployment-limitations-for-sno-ran_{context}"]
+== Limitations
+
+For deploying {sno}, LVM Storage has the following limitations:
+
+* The total storage size is limited by the size of the underlying Logical Volume Manager (LVM) thin pool and the overprovisioning factor.
+* The size of the logical volume depends on the size of the Physical Extent (PE) and the Logical Extent (LE).
+** It is possible to define the size of PE and LE during the physical and logical device creation.
+** The default PE and LE size is 4 MB.
+** If the size of the PE is increased, the maximum size of the LVM is determined by the kernel limits and your disk space.
+
+.Size limits for different architectures using the default PE and LE size
+[cols="1,1,1,1,1", width="100%", options="header"]
+|====
+|Architecture
+|RHEL 5
+|RHEL 6
+|RHEL 7
+|RHEL 8
+
+|32-bit
+|16 TB
+|16 TB
+|-
+|-
+
+|64-bit
+a|8 EB footnote:[Theoretical size.]
```

Comment:
Is this the proper format for footnotes in a table? It looks unusual to have the footnote in the cell (although it probably works better). The Doc Guidelines suggest an [alternative formatting for table cell footnotes](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables).